CHANGELOG.rst

```diff
@@ -34,6 +34,7 @@ Since last release
 
 **Changed:**
 
+* Fixed some bugs in facility_cost.cycpp.h and economic_entity.h (#1884)
```

Comment:
Note: it is sometime reasonable to collapse changes like this into a single line with multiple PR numbers.  E.g. in this case the lines with 1864, 1877, and 1870?